pkg/fission-cli/cmd/spec/list.go

```diff
@@ -381,72 +382,72 @@ func ShowAppliedKubeWatchers(ws []fv1.KubernetesWatchTrigger) {
 	}
 }
 
-// getAllFunctions get lists of functions in all namespaces
-func getAllFunctions(opts *ListSubCommand) ([]fv1.Function, error) {
-	fns, err := opts.Client().V1().Function().List("")
+// GetAllFunctions get lists of functions in all namespaces
+func GetAllFunctions(client client.Interface) ([]fv1.Function, error) {
```

Comment:
`GetXXXX` are only used in the spec package, let's keep these function as private functions. We can always expose them later once we import these functions in other packages.